rskj-core/src/main/resources/config/regtest.conf

```
@@ -12,7 +12,8 @@ blockchain.config {
         hop400 = 0,
         hop401 = 0,
         fingerroot500 = 0
-        tbd600 = 0
+        tbd600 = -1
```

```suggestion
        tbd600 = 0
```
should be 0 for regtest